client/src/components/Card/index.js

```diff
@@ -18,32 +18,31 @@ function Card(props) {
     restaurant: { name, price, district, id, cover, note, reviews, category },
     onClick
   } = props;
+  const coverURL = cover[0] ? cover[0].url : '';
 
   return (
     <StyledCard onClick={() => onClick(id)}>
       <div className="img-wrapper">
-        {cover.length && (
-          <Img
-            type="article"
-            src={`${process.env.REACT_APP_BACKEND_URL}${cover[0].url}`}
-            alt="cover"
-          />
-        )}
+        <Img
+          type="article"
+          src={`${process.env.REACT_APP_BACKEND_URL}${coverURL}`}
+          alt="cover"
+        />
       </div>
       <div className="card-infos">
         <div className="left-infos">
           <H4>{name}</H4>
           <p className="description">
             <Price value={price} />
             &nbsp;•&nbsp;
-            <span>{category.name}</span>
+            <span id="category-name">{category && category.name}</span>
```

Comment:
Does this id exist only for tests ?

Reply:
yes, but i Have updated the tests so it's no longer needed